tests/tp/test_tp_strategies.py

```diff
@@ -151,58 +149,50 @@ def get_loss_array(trainer: Trainer):
 @pytest.mark.world_size(4)
 @pytest.mark.parametrize('tp_degree', [2])
 @pytest.mark.parametrize('tp_strategy', ['ffn'])
-def test_tp_train(tp_degree: int, tp_strategy: str):
+def test_tp_train(tp_degree: int, tp_strategy: str, tmp_path: Path):
     """Test that we can train with FSDP-TP."""
-    my_dir = Path('/my-data-dir')
-
-    try:
-        # create c4 dataset
-        if my_dir.is_dir() and my_dir.exists():
-            shutil.rmtree(my_dir)
-        my_dir.mkdir(parents=True)
-        tp_dataset_name = create_c4_dataset_xxsmall(my_dir)
-
-        # Train model with TP and get loss
-        tp_cfg = get_cfg(pathlib.Path(tp_dataset_name), tp_strategy, tp_degree)
-        tp_trainer = train(tp_cfg)
-        tp_trainer.close()
-        tp_loss = get_loss_array(tp_trainer)
-
-        # Compare loss and expected loss for TP
-        import numpy as np
-        expected_tp_loss = np.array([
-            12.02126884,
-            11.96996498,
-            12.02957344,
-            11.97966957,
-            11.99677086,
-            11.96347618,
-        ])
-        np.testing.assert_allclose(tp_loss, expected_tp_loss)
-    except Exception as e:
-        raise e
-    finally:
-        # always remove the directory
-        if os.path.isdir(my_dir):
-            shutil.rmtree(my_dir)
+    from composer.utils import dist, get_device
+
+    dist.initialize_dist(get_device(None), timeout=10)
```

Comment:
dist should be already initialized by the autouse fixture

Reply:
ah i see. There are some other tests that initialize_dist, so we can remove those too?

Reply:
should be able to, yeah